solutions/python_azure_sdk/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.9-slim
+ARG TAG
+ARG PACKAGES
+
+RUN apt update && apt install -y git
+
+WORKDIR /
+RUN git clone -b ${TAG} https://github.com/Azure/azure-sdk-for-python --single-branch --depth 1
```

Comment:
I think you can clone the repo outside the container and then pass the folder as "-v" to individual docker build commands.

Reply:
I don't think `docker build` has the `-v` option. Do you mean to clone the repo on host and run mystikos with hostfs?

Reply:
I mean clone the source code on the host and then copy the files during "docker build". The point is that we don't have to clone the same repo 3 times.

Reply:
That's an interesting idea. But anyways, I took your advice to publish the docker image. So git clone doesn't matter now.